tutorials.md

```diff
@@ -17,19 +17,23 @@ Python.
 
 -   **Getting Started: Analyzing Census Data with Data Commons**
 
-    [Google Colab](https://colab.sandbox.google.com/drive/1wYohDirOgVxvmL0d-oJRWdD6AXfAX_w1){:target="_blank"}, [Github](https://github.com/datacommonsorg/api-python/blob/master/notebooks/analyzing_census_data.ipynb){:target="_blank"}
+    [Google Colab](https://colab.research.google.com/drive/1wYohDirOgVxvmL0d-oJRWdD6AXfAX_w1){:target="_blank"}, [Github](https://github.com/datacommonsorg/api-python/blob/master/notebooks/analyzing_census_data.ipynb){:target="_blank"}
+
+-   **COVID-19 Feature Exploration Analysis with Data Commons (by Google Health)**
```

Comment:
I am soliciting opinions on the title. Notice how all the other ones have
"Getting Started:" or "Case Study:"

Reply:
I like consistent naming :) How about:

  **Case Study: COVID-19 Feature Exploration Analysis (by Google Health)**

"with Data Commons" I feel is redundant and the ones below don't have it.

Reply:
Done, thanks for the opinion!